src/dsp_tools/commands/xmlupload/models/serialise/serialise_rdf_value.py

```diff
@@ -1,56 +1,44 @@
-from abc import ABC
-from abc import abstractmethod
+from __future__ import annotations
+
 from dataclasses import dataclass
 
-from rdflib import RDF
-from rdflib import BNode
-from rdflib import Graph
 from rdflib import Literal
 from rdflib import Namespace
 from rdflib import URIRef
 
 KNORA_API = Namespace("http://api.knora.org/ontology/knora-api/v2#")
 
 
-@dataclass(frozen=True)
-class ValueRDF(ABC):
-    resource_bn: BNode
-    prop_name: URIRef
+@dataclass
```

Comment:
This is a class to hold the transformed information, so that, the serialisation function does not have to worry about transformations.
Once we create the anti-corruption layer with the classes, this will no longer be necessary.